docker-compose.yml

```diff
@@ -1,12 +1,22 @@
 version: '3'
 services:
+
     web:
-        build: ./
+        build: ./.docker/web
         container_name: svgtranslate_web
```

Comment:
This isn't necessary, compose will give it a name for you. :)

Reply:
Removed both names.